lib/screens/EditNoteScreen.tsx

```diff
@@ -457,25 +443,22 @@ const EditNoteScreen: React.FC<EditNoteScreenProps> = ({
             style={{ flex: 1 }}
             ref={scrollViewRef}
           >
-            <RichEditor
-              ref={(r) => (richTextRef.current = r)}
-              style={[NotePageStyles().editor, {flex: 1, minHeight: 650 }]}
-              editorStyle={
-              {
-                contentCSSText: `
-                  position: absolute; 
-                  top: 0; right: 0; bottom: 0; left: 0;
-                `,
-                backgroundColor: theme.primaryColor,
-                color: theme.text,
-              }}
-              autoCorrect={true}
-              placeholder="Write your note here"
-              onChange={(text) => setText(text)}
-              initialContentHTML={text}
-              onCursorPosition={handleScroll}
-              disabled={!owner}
-            />
+         <View style={[NotePageStyles().richTextContainer]}>
+          <RichText
+            editor={editor}
+            placeholder="Write something..."
+            style={[NotePageStyles().editor, {backgroundColor: Platform.OS == "android" && "white"}]}
+          />
+          </View>
+       
+        {/* Toolbar placed at the bottom */}
+        <View style={[NotePageStyles().toolBar]}>
+          <Toolbar
+            editor={editor}
+            style={NotePageStyles().container}
+            actions={['bold', 'italic', 'underline', 'bullet_list', 'blockquote', 'indent', 'outdent', 'close_keyboard' ]}
+          />
+        </View>
```

Comment:
Nice addition to the code and watching out for anything that needed to be added.